content/docs/1.3.0/references/networking.md

```diff
@@ -28,8 +28,9 @@ From | Port | Protocol
 To | Port | Protocol
 --- | --- | ---
 `Other Longhorn Manager` | 9500 | TCP
-`Instance Manager` |  8500; 10000-30000 | TCP
-`Backing Image Manager` |  8000; 8001 | TCP
+`Instance Manager` | 8500; 8501; 10000-30000 | TCP
```

Comment:
Will Longhorn managers directly talk with engine/replica processes after introducing the proxy? @c3y1huang

Reply:
Is the port `8501` for proxy grpc calls from longhorn-manager to the instance manager?

Reply:
Yes. I missed this update.

Reply:
> Will Longhorn managers directly talk with engine/replica processes after introducing the proxy? @c3y1huang

Longhorn managers still manage processes through the 8500 port.

Reply:
How about port 10000-30000? Can we remove this? (Although I already removed this from the instance manager ingress part.)

Reply:
I am not entirely sure. Proxy is not responsible for handling tgt processes and their connections. And as far as I know, the process connections are to the local iSCSId from the IM-e, so I am not sure why the Manager needs to have those ports on egress.

Reply:
longhorn managers will handle the instance managers via 8500. As for the tgt part, it's the responsibility of the engine processes inside instance managers. I think we can remove it first and see the test result.